File: swift/stringTest.swift

func test() {
	let a = "apple"
	print(a, a.count)
}

func test2() {
	let a = "apple"
	for c in a {
		print(c)
	}
}

func test3() {
	let a = "apple"
	for c in a.utf8 {
		print(c)
	}
}

File: c/stringTest.c
#include <stdio.h>

void test(){
	char a[]="apple";
	printf("%s\n", a);
}

void test2(){
	char a[10]="apple";
	printf("%s\n", a);
}

void test3(){
	char a[10]={'a', 'p', 'p', 'l', 'e', '\0'};
	printf("%s\n", a);
}

void test4(){
	char *a="apple";
	printf("%s\n", a);
}

void main(){
	test();
}